README.md

```diff
@@ -398,6 +398,30 @@ setTimeout(function() {
   proxy.removeResolver(topPriority);
 }, 600000);
 ```
+### Modify Request body
+
+bodyChange options parameter let you modify the incoming req.body. An example is below:
```

Comment:
I think a better name would be just ```transform``` since it is already inside an option called "body" it is an unnecessary prefix name, and "transform" describes better the intent than "change"